polyfill/test/intl.mjs

```diff
@@ -59,6 +59,7 @@ describe('Intl', () => {
   const at = new Intl.DateTimeFormat('de-AT', { timeZone: 'Europe/Vienna' });
 
   describe('should work for Absolute', () => {
+    // TODO why is the month different for DateTime for de-AT
```

Comment:
Not sure I understand this comment? The month looks the same in both cases to me.

Reply:
Good catch. This comment crept in while the issue was resolved. Removed.